### PR TITLE
buckshot buff - 60 to 75 on all pellet pointblank

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
see title, accomplishes this by bumping individual pellet starting damage from 10 to 12.5
## Why It's Good For The Game
giving a reason to pb the shit out of dudes with buck instead of the all-rounder slug
## Changelog
:cl:
balance: Buckshot individual pellet damage up from 10 to 12.5. Still firing 6 pellets.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
how much damage falloff do they got doe 

i actually don't know how bad damage falloff is on shotguns because i just always pb 😂

**Comment by Hatterhat** [comment]
still the default damage falloff

**Comment by ghost** [comment]
i don't know the damage falloff bro

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 1]
```diff
@@ -61,7 +61,7 @@
 
 /obj/item/projectile/bullet/pellet/shotgun_buckshot
 	name = "buckshot pellet"
-	damage = 10
+	damage = 13.5
```
Comment (Ghommie) [inline code/modules/projectiles/projectile/bullets/shotgun.dm]: But the original damage was:
```suggestion
	damage = 12.5
```

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Comment by silicons** [comment]
@Hatterhat outright implement requiring a free hand to rack shotguns/being able to rack them through alt click if they are on floor if you're doing this.

**Comment by Ghommie** [comment]
@kevinz000 tgstation made riot/combat shotguns heavy firearms (ergo need two hands to use them) while giving sawn off shotties less accuracy and extra recoil iirc. That's a better approach to be honest.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**